files/debian-initd

```diff
@@ -180,24 +180,31 @@ do_reload() {
 #
 log_end_msg_wrapper()
 {
+	if [ $1 != 0 ] && [ $1 != $2 ]; then
+		value=1
+	else
+		value=0
+	fi
 	if [ "$3" != "no" ]; then
-		[ $1 -lt $2 ] && value=0 || value=1
 		log_end_msg $value
 	fi
+	if [ $code != 0 ]; then
```

Comment:
This gives an error for me: `[: !=: unexpected operator`. Quoting the args fixes it for me, though:
```
if [ "$code" != "0" ]; then
```

Reply:
oh oh... Should have been "$value" not "$code" (was thinking about "exit code" too much I guess)
(since then I usually `set -u` in my scripts, but didn't want to do that here and here you go -- can't even see what I am typing), but should be ok to wrap "0" as well

thanks @Uwila , pushing the fix...